plugins/micromega/simplex.ml

```diff
@@ -18,6 +18,44 @@ type ('a, 'b) sum = Inl of 'a | Inr of 'b
 
 let debug = false
 
+(** Exploiting profiling information *)
+
+let profile_info = ref []
+let nb_pivot = ref 0
```

Comment:
Is this synchronized with the document state?  I think you might want to use `Summary.ref` here or something like that to make this play nicely with backtracking?